spaceship/templates/register.html

```diff
@@ -3,7 +3,8 @@
 {% block title %}Register - {{ super() }}{% endblock %}
 
 {% block content %}
-<h3 class="text-center">Spaceship Earth needs you to become a captain!</h3>
+<h3 class="text-center">Spaceship Earth needs you to become a Team Captain!</h3>
+<h5 class="text-center">(so you can invite more friends to join our crew)</h5>
```

Comment:
this seems like a very "us focused" reason.  what about "so you can help your friends take action on climate change"